source/stylesheets/old-all.css

```diff
@@ -1,29 +0,0 @@
-#info {
-  text-align: right;
-}
```

Comment:
We didn't pull this over, but it looks like it might no longer be used.  Or it's a typo for `.info` which is used occasionally.  On reflection though, I'm not sure I'd want the things with `info` class to be right aligned.

tl;dr - dropping this style seems fine.

Reply:
Awesome! I wasn't sure what to do with this one, I landed on just removing it as it wasn't being used :)